src/store/actions/keymap.ts

```diff
@@ -0,0 +1,72 @@
+import { Injectable } from '@angular/core';
+import { Action } from '@ngrx/store';
+
+import { Keymap } from '../../config-serializer/config-items/Keymap';
+
+@Injectable()
+export class KeymapActions {
+    static PREFIX = '[Keymap] ';
+
+    static ADD = KeymapActions.PREFIX + 'Add keymap';
+    static DUPLICATE = KeymapActions.PREFIX + 'Duplicate keymap';
+    static EDIT_ABBR = KeymapActions.PREFIX + 'Edit keymap abbreviation';
+    static EDIT_NAME = KeymapActions.PREFIX + 'Edit keymap title';
+    static SAVE_KEY = KeymapActions.PREFIX + 'Save key action';
+    static SET_DEFAULT = KeymapActions.PREFIX + 'Set default option';
+    static REMOVE = KeymapActions.PREFIX + 'Remove keymap';
+
+    addKeymap(item: Keymap): Action {
+        return {
+            type: KeymapActions.ADD,
+            payload: item
+        };
+    }
+
+    setDefault(abbr: string): Action {
```

Comment:
I don't think so. Because this functions are not doing the same thing, so there can't be really and consistency. I would strongly stay with this approach.